docs/src/testing/testing.md

```diff
@@ -36,6 +36,11 @@ Tests: 1 passed, 0 failed, 0 skipped
 Forge considers all functions in your project marked with `#[test]` attribute as tests.
 Test functions cannot return any values and cannot take any arguments.
 
+Starknet Forge will collect tests only from these places:
+
+- any files reachable from the package root (declared as `mod` in `lib.cairo` or its children)
+- files in directories outside the main package
```

Comment:
```suggestion
- files in all directories outside of the main package
```
all or tests dir only?

Reply:
It's all files outside of the package actually. In directories or not.